spec.emu

```diff
@@ -15,16 +15,13 @@ copyright: false
   <emu-alg>
     1. If _iterables_ is not an Object, throw a *TypeError* exception.
     1. Set _options_ to ? GetOptionsObject(_options_).
-    1. Let _longest_ be ? GetOption(_options_, *"longest"*, ~boolean~, ~empty~, *false*).
-    1. Let _strict_ be ? GetOption(_options_, *"strict"*, ~boolean~, ~empty~, *false*).
-    1. If _longest_ is *true* and _strict_ is *true*, throw a *TypeError* exception.
-    1. Let _mode_ be ~shortest~.
+    1. Let _mode_ be ? Get(_options_, *"mode"*).
+    1. If _mode_ is not one of *"shortest"*, *"longest"*, *"strict"*, or *undefined*, throw a *TypeError* exception.
+    1. If _mode_ is *undefined*, set _mode_ to *"shortest"*.
```

Comment:
I did these two steps in the other order in [the base64 proposal](https://tc39.es/proposal-arraybuffer-base64/spec/#sec-uint8array.prototype.tobase64). No observable difference, just editorial.

```suggestion
    1. If _mode_ is *undefined*, set _mode_ to *"shortest"*.
    1. If _mode_ is not one of *"shortest"*, *"longest"*, or *"strict"*, throw a *TypeError* exception.
```

Reply:
Okay.